RHIDP-2376 Document which plugins are recommended for Prod Readiness (#310)
redhat-developer/red-hat-developers-documentation-rhdh: artifacts/snip-plugins-production-readiness.adoc

[id='snip-plugins-production-readiness_{context}']
[NOTE]
====

> The following plugins have currently been vetted for RBAC/Audit logging capabilities: 

Reviewer: ```suggestion
The following plugins are currently vetted for RBAC and audit logging capabilities: 
```
Just because it sounds weird to use the word "currently" with "have been" 
Also not sure about the word "vetted" wrt translation, but nothing advised against it.

Reviewer 2: how about, just simpler: 

```
The following plugins support  RBAC and audit logging capabilities: 
```

> * RBAC

Reviewer 2: so the RBAC plugin has RBAC support? this seems circular...

Author: Updated text resolves this circular reference.

Reviewer 3: This topic is not needed anymore, please remove

* OCM
* Topology
* Keycloak
* Quay Actions

> You should consider your security hardening requirements before you use plugins that are not yet vetted. 

Reviewer 2: ```suggestion
You should consult with your organization's security experts before using any plugins that do not support authentication and logging.
```
